frontend/src/views/WheresMyGeneV2/components/HeatMap/index.tsx

```diff
@@ -476,26 +416,21 @@ export default memo(function HeatMap({
             {allTissueCellTypes.map(
               ({ tissueId, tissueName, tissueCellTypes }) => {
                 return (
-                  tissueCellTypes.length > 0 && (
```

Comment:
`tissueCellTypes.length > 0` remove this guard clause since `allTissueCellTypes` no longer contains tissues with 0 cell types